src/include/rear/ui.rb

```diff
@@ -68,6 +68,23 @@ def UsedModules
       deep_copy(modules)
     end
 
+    # returns a list of mountpoints that have to be included in BACKUP_PROG_INCLUDE
+    def GetMountpoints
+      # -D excludes pseudo filesystems
+      # -r raw output, no ascii-art
+      # -n no column header
+      cmd = "findmnt -D -t notmpfs,nodevtmpfs -o TARGET -n -r | sort"
```

Comment:
-t --types is listed the line after -T and -M
I know about the json output, but the solution I chose is way easier to parse.